main/src/library/Fixpoint/Interpreter.flix

```diff
@@ -92,15 +92,15 @@ mod Fixpoint {
                 let rel = MutMap.getOrElsePut!(ramSym, MutMap.new(r1), db);
                 match toDenotation(ramSym) {
                     case Denotation.Relational =>
-                        let tuple = List.toVector(List.map(evalTerm(env), terms));
+                        let tuple = Vector.map(evalTerm(env), terms);
                         MutMap.put!(tuple, Reflect.default(), rel)
                     case Denotation.Latticenal(bot, leq, lub, _) =>
                         // assume that length(terms) > 0
-                        let len = List.length(terms);
-                        let keyList = terms |> List.map(evalTerm(env));
-                        let (relKeys, latValList) = List.splitAt(len-1, keyList);
-                        let key = List.toVector(relKeys);
-                        let latVal = match List.head(latValList) {
+                        let len = Vector.length(terms);
```

Comment:
Is this because Vector lacks a `splitAt`?

Reply:
Yes

Reply:
Sure!

Reply:
Ok, let us do that and then rebase.